src/functions.php

```diff
@@ -16,7 +16,7 @@ function displayDreamDetails(array $dreams): string {
             . "<p>" . $dream['dream_description'] . "</p>";
 
         } else {
-            throw new Exception('Invalid array keys');
+            throw new Exception('<p>Invalid array keys</p>');
```

Comment:
if you use this in the catch "error_log($exception->getMessage() . "\n", 3, 'serverlog.log'); ""

Reply:
Done! Resolved